src/Prismic/Predicates.php

```diff
@@ -8,16 +9,18 @@
  * A set of helpers to build predicates
  * @package Prismic
  */
-class Predicates {
+class Predicates
+{
 
     /**
      * @param string $fragment
      * @param string $value
      *
      * @return SimplePredicate
      */
-    public static function at($fragment, $value) {
-        return new SimplePredicate("at", $fragment, array($value));
+    public static function at($fragment, $value)
```

Comment:
types? (for all predicates)

Reply:
yep for each param and the return type. You did that everywhere and it's really cool, why not here?

Reply:
This pull took so long to get review, I abandoned it and started working on my own fork. The fork referenced above has a whole load of other stuff not present in this PR, such as hydrating result sets into concrete objects, so you can type hint for your own `\Acme\BlogPost` or whatever and then use the content model to alleviate the constant "is it null or not" that you end up doing in views